go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 require (
 	cloud.google.com/go/kms v1.15.3
 	cloud.google.com/go/storage v1.33.0
-	filippo.io/age v1.1.1
+	filippo.io/age v1.1.2-0.20230920124100-101cc8676386
```

Comment:
I guess a new age release would be still a blocker for this, but review could already happen before that, so by the time we have an age update, this can be merged.